code/square_root_decomposition/src/mos_algorithm/sqrtdecomposition.py

```diff
@@ -0,0 +1,55 @@
+# PROBLEM STATEMENT-> You are given an array A of size N and Q queries.(All indexes are 1 based).
+# Each query can be of two types
+#       1->   1 i  - add 1 to the ith element of the array A
+#       2->   2 K  - print the Kth odd number of the array A if it exists, else print -1
+
+import math
+n, q = list(map(int, input().strip().split(' ')))
```

Comment:
This should all go under an `if __name__ == '__main__':`.

Reply:
OK sir got it.

Reply:
Done Sir! please review my latest PR  https://github.com/OpenGenus/cosmos/pull/4663